src/components/Asset.js

```diff
@@ -12,36 +12,340 @@ const Div = styled.div`
 `;
 
 export default function Asset() {
-  const { asset } = useSelector((state) => state.user.user);
+  const ws = new WebSocket(process.env.REACT_APP_WEBSOCKET_SERVER_URL);
 
-  const ownedCash = asset.cash;
+  const { asset } = useSelector((state) => state.user.user);
+  const { transactionHistory } = useSelector((state) => state.user.user);
+  const [newCoinList, setNewCoinList] = useState([]);
   const ownedCoinList = asset.coins;
 
-  const getCurrentLeftMoney = (coin) => {
-    const leftMoney =
-      Number(ownedCash) - Number(coin.averagePrice * coin.quantity);
-    return leftMoney;
+  const [socketData, setSocketData] = useState("");
+  const [renderedAssetList, setRenderedAssetList] = useState({});
+  const [isSortBtnClick, setIsSortBtnClick] = useState(false);
+  const [isAscendSortByName, setIsAscendSortByName] = useState(true);
+  const [isAscendSortByLeftMoney, setIsAscendSortByLeftMoney] = useState(true);
+  const [isAscendSortByAvgPrice, setIsAscendSortByAvgPrice] = useState(true);
+  const [isAscendSortByBoughtPrice, setIsAscendSortByBoughtPrice] =
+    useState(true);
+  const [isAscendSortByEvaluatedPrice, setIsAscendSortByEvaluatedPrice] =
+    useState(true);
+  const [isAscendSortByEvaluatedProfit, setIsAscendSortByEvaluatedProfit] =
+    useState(true);
+  const [isAscendSortByYieldRate, setIsAscendSortByYieldRate] = useState(true);
```

Comment:
```suggestion
  const [isAscendSort, setIsAscendSort] = useState({
    isAscendName = true,
    isAscendLeftMoney = true,
    ~~~
  })
```
현재 모두 오름차순이 되어 있는지 안되어 있는지를 boolean 값으로 하나하나 관리하시는 거 같아요
제가 생각했을 때는 위와 같이 객체로 관리할 수 있을 거 같습니다.! 그럼 훨씬 깔끔하고 가독성도 좋아질 거 같아요

Reply:
저도 이방식이 좋아보이네요!

Reply:
적용했습니다!